### DISCUSSION TITLE
What happened to jokaorgua fork?

### DISCUSSION THREAD

**Discussion by tboy1337** [masterking32/MasterHamsterKombatBot#99] [General]
hey @jokaorgua what happened to your fork where you moved the accounts into a different .py file?  Are you going to change it back or can you provide me with the import section on how you did it?  I remember it was just one line but i cant remember what it was, thanks.

**Comment by jokaorgua** [comment]
I’ve closed that PR because it has not been accepted. 
If you want to move accounts list to a separate file just move the AccountList array to file account_list.py and in main.py write 
‘Import AccountList from account_list’

**Comment by tboy1337** [comment]
Thanks xD

**Comment by tboy1337** [comment]
just got it working and i found out that its 'from account_list import AccountList' instead of 'Import AccountList from account_list'.  Maybe thats why the PR was denied?

**Comment by tboy1337** [comment]
Resolved
